robocorp-code/src/robocorp_code/inspector/inspector_api.py

```diff
@@ -721,49 +918,95 @@ def cancel_lint(self, *args, **kwargs):
+            log.debug("Man:: Killed Image Inspector!")
+
     ####
     #### ENQUEUE COMMANDS TO THREAD WORKERS
     ####
     def _enqueue_web(self, cmd: _WebBaseCommand, wait: bool):
-        self._web_inspector_thread.queue.put(cmd)
-        if wait:
-            cmd.handled_event.wait(20)
-            try:
-                return cmd.future.result()
-            except Exception as e:
-                return {"success": False, "message": str(e), "result": None}
+        if self._web_inspector_thread:
+            self._web_inspector_thread.queue.put(cmd)
+            if wait:
+                cmd.handled_event.wait(20)
+                try:
+                    return cmd.future.result()
+                except Exception as e:
+                    return {"success": False, "message": str(e), "result": None}
+
         return {"success": True, "message": None, "result": None}
 
     def _enqueue_windows(
         self, cmd: _WindowsBaseCommand, wait: bool = True
     ) -> ActionResultDict:
-        self._windows_inspector_thread.queue.put(cmd)
-        if wait:
-            try:
-                return cmd.future.result()
-            except Exception as e:
-                return {"success": False, "message": str(e), "result": None}
+        if self._windows_inspector_thread:
+            self._windows_inspector_thread.queue.put(cmd)
+            if wait:
+                try:
+                    return cmd.future.result()
+                except Exception as e:
+                    return {"success": False, "message": str(e), "result": None}
 
         return {"success": True, "message": None, "result": None}
 
     def _enqueue_image(
         self, cmd: _ImageBaseCommand, wait: bool = True
     ) -> ActionResultDict:
-        self._image_inspector_thread.queue.put(cmd)
-        if wait:
-            try:
-                return cmd.future.result()
-            except Exception as e:
-                return {"success": False, "message": str(e), "result": None}
+        if self._image_inspector_thread:
+            self._image_inspector_thread.queue.put(cmd)
+            if wait:
+                try:
+                    return cmd.future.result()
+                except Exception as e:
+                    return {"success": False, "message": str(e), "result": None}
+
+        return {"success": True, "message": None, "result": None}
+
+    def _enqueue_java(
+        self, cmd: _JavaBaseCommand, wait: bool = True
+    ) -> ActionResultDict:
+        if self._java_inspector_thread:
+            self._java_inspector_thread.queue.put(cmd)
+            if wait:
+                try:
+                    return cmd.future.result()
+                except Exception as e:
+                    return {"success": False, "message": str(e), "result": None}
 
         return {"success": True, "message": None, "result": None}
 
     ####
     #### WEB RELATED APIs
     ####
+    @log_call(log_level=LogLevel.DEBUG)
```

Comment:
I meant that these calls should be removed (I think that when you enable debug mode you now have 2 mentions whenever you have an invocation, one showing the message that this was requested and this message directly, so, it should be redundant).

Reply:
OK. I'll remove it.